legal-api/src/legal_api/models/db.py

```diff
@@ -47,27 +42,9 @@ class Transaction(db.Model):
                 autoincrement=True
             )
     remote_addr = db.Column(db.String(50), nullable=True)
-    issued_at = db.Column(db.DateTime, default=datetime.utcnow, nullable=True)
-
-
-def print_versioning_info():
-    """
-    Print the current versioning status if not already printed.
-
-    This should only be called within an application context.
-    """
-    try:
-        from legal_api.services import flags as flag_service  # pylint: disable=import-outside-toplevel
-
-        current_service = current_app.config.get('SERVICE_NAME')
-        if current_service:
-            db_versioning = flag_service.value('db-versioning')
-            use_new_versioning = (bool(db_versioning) and bool(db_versioning.get(current_service)))
-            current_versioning = 'new' if use_new_versioning else 'old'
-            current_app.logger.info(f'\033[31mService: {current_service}, db versioning={current_versioning}\033[0m')
-    except Exception as err:
-        # Don't crash if something goes wrong
-        current_app.logger.error('Unable to read flags: %s' % repr(err), exc_info=True)
+    issued_at = db.Column(db.DateTime,
+                          default=datetime.now(timezone.utc),
```

Comment:
`default` needs a callback function to get the current datetime. we might need a custom function to support this